packages/autorest.python/samples/specification/directives/generated/azure/directives/sample/aio/_operations/_operations.py

```diff
@@ -10,8 +10,9 @@
 import sys
 from typing import Any, AsyncIterable, Callable, Dict, IO, Optional, TypeVar, Union, cast, overload
 
-from my.library.aio import AsyncCustomDefaultPollingMethod, AsyncCustomPager, AsyncCustomPoller, AsyncList
+from my.library.aio import AsyncCustomDefaultPollingMethod, AsyncCustomPager, AsyncCustomPoller
 
+from azure.core.async_paging import AsyncList
```

Comment:
this fix is correct, but I'm not seeing any code change that resulted in it, was this manually changed?